docker/rakyll-hey/verify.py

```diff
@@ -0,0 +1,14 @@
+from subprocess import PIPE, Popen
+from io import StringIO
+import pandas as pd
+
+p = Popen(['hey'], stdout=PIPE, stderr=PIPE, universal_newlines=True)
```

Comment:
I would use subprocess.check_output instead of Popen. It makes the code easier to understand: https://docs.python.org/3/library/subprocess.html#subprocess.check_output